File: getUsers/handler.js
const aws = require("aws-sdk");

const dynamodb = new aws.DynamoDB.DocumentClient({});

const getUsers = async (event, context) => {
  let userId = event.pathParameters.id;

  const params = {
    TableName: "usersTable",
    KeyConditionExpression: "pk = :pk",
    ExpressionAttributeValues: {
      ":pk": userId,
    },
  };

  return dynamodb
    .query(params)
    .promise()
    .then((res) => {
      console.log(res);
      return {
        statusCode: 200,
        body: JSON.stringify({ user: res.Items }),
      };
    });
};

module.exports = {
  getUsers,
};

File: deleteUsers/handler.py
import boto3, json

client = boto3.resource('dynamodb')

table = client.Table('usersTable')

def deleteUsers(event, context):
    user_id = event['pathParameters']['id']
    result = table.delete_item(Key = {'pk': user_id})

    body = json.dumps({ 'message' : f"user {user_id} deleted" })

    response = {
      'statusCode': result['ResponseMetadata']['HTTPStatusCode'],
      'body': body
    }

    return response